lib/phoenix/live_dashboard/pages/ecto_stats_page.ex

```diff
@@ -57,7 +118,34 @@ defmodule Phoenix.LiveDashboard.EctoStatsPage do
 
   @impl true
   def render_page(assigns) do
-    nav_bar(items: items(assigns))
+    if assigns[:error] do
+      render_error(assigns)
+    else
+      items =
+        for repo <- assigns.repos do
+          info_module = info_module_for(repo)
+
+          {repo,
+           name: format_nav_name(repo),
+           render: fn ->
+             render_repo_tab(%{
+               repo: repo,
+               info_module: info_module,
+               ecto_options: assigns.ecto_options
+             })
+           end}
+        end
+
+      nav_bar(items: items, nav_param: :repo, extra_params: ["nav"])
+    end
+  end
+
+  defp render_repo_tab(assigns) do
+    nav_bar(items: items(assigns), extra_params: ["repo"])
+  end
+
+  defp format_nav_name(repo) do
+    "#{repo |> inspect() |> String.replace(".", " ")} Stats"
```

Comment:
I think we can keep the repo name as is now and remove the stats part, specially given the page is already called Ecto Stats:

```suggestion
    inspect(repo)
```